docs/pages/build-reference/easignore.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Ignoring files
+description: Learn how to configure EAS to ignore unnecessary files during the build process.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+A `.easignore` file defines which files [EAS](https://expo.dev/eas) should ignore when uploading your project to the [EAS Build](/build/introduction) servers.
+
+> **info** Ignoring unnecessary files can help reduce your app's archive size and upload time.
+
+By default, the [EAS CLI](/build/setup/#install-the-latest-eas-cli) refers to the [`.gitignore`](https://git-scm.com/docs/gitignore) file (if it exists) to determine which files to ignore. If you create a `.easignore` file, the EAS CLI prioritizes it over the `.gitignore` file. When creating a `.easignore` file, include all files from your `.gitignore` file, and add additional files you want to ignore.
+
+## Getting started with .easignore
+
+<Step label="1">
+
+Create a `.easignore` file in the root of your project.
+
+</Step>
+
+<Step label="2">
+
+Copy the content of the `.gitignore` file into the `.easignore` file. Then, add any files that are unnecessary for the build process.
+
+```bash .easignore
+# Copy everything from your .gitignore file here
+
+# Additional ignores specific to EAS builds
+
+# Ignore native folders (If you are using CNG)
+/ios
+/android
```

Comment:
```suggestion
/android
/ios
```

Let's also use the [correct platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms).